src/lib/draw-and-pick.js

```diff
@@ -338,34 +341,20 @@ function getPickedColors(gl, {
     // Clear the frame buffer
     gl.clear(GL.COLOR_BUFFER_BIT | GL.DEPTH_BUFFER_BIT);
 
-    // Set blend mode for picking
-    // always overwrite existing pixel with [r,g,b,layerIndex]
-    const settings = {
-      blend: true,
-      blendFunc: [gl.ONE, gl.ZERO, gl.CONSTANT_ALPHA, gl.ZERO],
-      blendEquation: gl.FUNC_ADD
-    };
-
-    withParameters(gl, settings, () => {
-      // Render all pickable layers in picking colors
-      layers.forEach((layer, layerIndex) => {
-        if (!layer.isComposite && layer.props.visible && layer.props.pickable) {
-
-          // Encode layerIndex with alpha
-          setParameters(gl, {blendColor: [0, 0, 0, (layerIndex + 1) / 255]});
-          layer.drawLayer({
-            moduleParameters: Object.assign({}, layer.props, {
```

Comment:
the settings used in this withParameters() has been accommodated by the outer withParameters() call at line 328